avocado/core/loader.py

```diff
@@ -233,7 +233,7 @@ def get_type_label_mapping(self):
     def get_decorator_mapping(self):
         return self._decorator_mapping
 
-    def discover(self, references, which_tests=DEFAULT):
+    def discover(self, references, which_tests=DEFAULT, force=None):
```

Comment:
Since the other parameters are documented, it's a good idea to also document this one.